packages/@guardian/source-react-components/src/brand/SvgGuardianBestWebsiteLogo.tsx

```diff
@@ -0,0 +1,72 @@
+import { css } from '@emotion/react';
+import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { visuallyHidden } from '@guardian/source-foundations';
+
+export type SvgGuardianBestWebsiteLogoProps = {
+	/**
+	 * The text colour.
+	 *
+	 * This should probably come from
+	 * [`@guardian/src-foundation`](https://theguardian.design/2a1e5182b/p/2668c8-code).
+	 */
+	textColor?: string;
+
+	/**
+	 * The text accent colour.
+	 *
+	 * This should probably come from
+	 * [`@guardian/src-foundation`](https://theguardian.design/2a1e5182b/p/2668c8-code).
+	 */
+	textAccentColor?: string;
+	/**
+	 * The width the SVG will display at (height is automatically adjusted
+	 * to maintain the correct aspect ratio).
+	 */
+	width?: number;
+};
+
+/**
+ * [Storybook](https://guardian.github.io/source/?path=/docs/packages-source-react-components-svgguardianbestwebsitelogo--playground) •
+ * [GitHub](https://github.com/guardian/source/tree/main/packages/@guardian/source-react-components/src/brand/SvgGuardianBestWebsiteLogo.tsx) •
+ * [NPM](https://www.npmjs.com/package/@guardian/source-react-components)
+ *
+ * The Guardian Best Website logo.
+ */
+
+export const SvgGuardianBestWebsiteLogo = ({
+	textColor,
+	textAccentColor,
+	width,
+}: SvgGuardianBestWebsiteLogoProps): EmotionJSX.Element => {
+	return (
+		<>
+			<svg
+				xmlns="http://www.w3.org/2000/svg"
+				viewBox="0 0 300 116"
```

Comment:
I'm a big fan of ratios that are shared, so would prefer 300&times;120 here.

https://www.mxdvl.com/tools/factors